core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/kafka/IngestFromKafkaDriver.java

```diff
@@ -238,88 +218,101 @@ public void consumeFromTopic(
 			throw new Exception(
 					"Kafka consumer connector is null, unable to create message streams");
 		}
-		final Map<String, Integer> topicCount = new HashMap<>();
-		topicCount.put(
-				formatPluginName,
-				1);
-
-		final Map<String, List<KafkaStream<byte[], byte[]>>> consumerStreams = consumer.createMessageStreams(topicCount);
-		final List<KafkaStream<byte[], byte[]>> streams = consumerStreams.get(formatPluginName);
-		int counter = 1;
-		for (final KafkaStream stream : streams) {
-			final ConsumerIterator<byte[], byte[]> it = stream.iterator();
-			queue.remove(formatPluginName);
+		try {
 			LOGGER.info("Kafka consumer setup for format [" + formatPluginName + "] against topic [" + formatPluginName + "]");
```

Comment:
May want to set this statement back to debug, or change the comment and move it to within the batch flush statement to show ingest progress / activity.